fix(curriculum): remove first person language from colored markers (#67406)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-colored-markers/617bbb6b97a83f6d1f7d6e38.md

# --description--

> Now that you've practiced with secondary colors, it is time to review how to create <dfn>tertiary colors</dfn>. Tertiary colors are created by combining a primary color with a nearby secondary color.

Reviewer: ```suggestion
Now that you've practiced with secondary colors, here's a review of how to create <dfn>tertiary colors</dfn>: tertiary colors are created by combining a primary color with a nearby secondary color.
```
